windows/src/support/texteditor/Editor.cpp

```diff
@@ -0,0 +1,221 @@
+/*
```

Comment:
The [code style](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide) is a bit wonky -- it should be 2 space tabstops, `{` at end of line, not next line, etc.